test/system/260-kube.bats

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests generated configurations for kube.
+#
+
+load helpers
+
+SERVICE_NAME="podman_test_$(random_string)"
+
+function setup() {
+    basic_setup
+}
+
+function teardown() {
+    basic_teardown
+}
+
+@test "podman kube generate - basic" {
+    run_podman kube generate --help
+    is "$output" ".*podman.* kube generate \[options\] {CONTAINER...|POD...|VOLUME...}"
+    run_podman generate kube --help
+    is "$output" ".*podman.* generate kube \[options\] {CONTAINER...|POD...|VOLUME...}"
+}
```

Comment:
I would suggest moving this test to `test/system/700-play.bats` as this is where all the other `kube` related tests are currently located.